### PR TITLE
Hunterised 1.74.2

### PR DESCRIPTION
**Pull request by xsacha**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -41,20 +47,43 @@ set_property(GLOBAL PROPERTY PREDEFINED_TARGETS_FOLDER "CMake Targets")
...
+set(VERSION_MINOR 74)
+set(VERSION_PATCH 2)
+set(VERSION_PLAIN ${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH})
+
+option(BUILD_PROG "Build utility programs" OFF)
+
+if(NOT EXISTS ${PROJECT_SOURCE_DIR}/.cppan)
+    find_package(GIF)
+    hunter_add_package(Jpeg)
+    find_package(JPEG)
```
Comment (ruslo) [inline CMakeLists.txt]: Please check wiki for usage instructions. E.g. it should be `find_package(JPEG CONFIG REQUIRED)` for Jpeg:
* https://github.com/ruslo/hunter/wiki/pkg.jpeg

**Review by ruslo — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (xsacha) [reply CMakeLists.txt]: Thanks,
I'll fix that up

**Comment by xsacha** [comment]
This one works now.
Thanks.

**Review by ruslo — APPROVED** [review]

**Comment by ruslo** [comment]
Released:
* https://github.com/hunter-packages/leptonica/releases/tag/v1.74.2-p0

**Pull request merged**